python/cac_tripplanner/destinations/views.py

```diff
@@ -130,3 +133,24 @@ def get(self, request, *args, **kwargs):
             results = results[:limit_int]
         data = serializers.serialize('json', results, fields=output_fields)
         return HttpResponse(data, 'application/json')
+
+class FeedEvents(View):
+    """ API endpoint for the FeedEvent model """
+
+    def get(self, request, *args, **kwargs):
+        """ GET 20 most recent feed events that are published
+
+        TODO: Additional filtering, dynamic limits?
```

Comment:
I think we may have differing thoughts on what is supposed to happen with these feed events. I was under the impression that they'd show up in the search results alongside the featured locations, and then they'd be selectable as a routing option. Would be good to have @flibbertigibbet and/or @jbranigan weigh in to make sure we implement this correctly.